### PR TITLE
feat: add map

### LINKED ISSUE #2: Get GeoJSON into repo and document its source

### LINKED ISSUE #7: Port the choropleth from Observable to outside Observable

### PR DESCRIPTION
**Pull request by ejfox**
Closes #2 by adding GeoJSON to `/public/` folder
Closes #7 by recreating the Observable Notebook choropleth

### CODE REVIEW

**Comment by ejfox** [comment]
GeoJson loaded and rendering

Screenshot 2023-02-02 at 11 06 05 AM

**Comment by ejfox** [comment]
Screenshot 2023-02-02 at 11 37 34 AM

Rendering data from `/data/ovuzpsg_1221/cleaned/all_data.json` created by the data processing script - right now everything is grouped together, need to add some UX/UI to navigate between different grain types, I think

**Comment by ejfox** [comment]
Screenshot 2023-02-02 at 1 56 24 PM

File: pages/map.vue [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+      
+        <thead>
+          <tr>
+            <th>Oblast</th>
+            <th>Harvested Area</th>
+            <th>Yield</th>
+            <th>Volume</th>
+          </tr>
+        </thead>
+        <TransitionGroup name="table" tag="tbody" v-if="dataByGrainType">
```
Comment (jeffwklein) [inline pages/map.vue:49]: total nitpick and not a blocker but should we run prettier on this?

**Review by jeffwklein — APPROVED** [review]

**Comment by curran** [comment]
@ejfox I don't think the `cursor:pointer` change landed in this, for the buttons.

**Pull request merged**